docs/source/cross_validation.rst

```diff
@@ -4,9 +4,15 @@ Cross Validation
 See the `scikit-learn cross validation documentation`_ for a fuller discussion of cross validation.
 This document only describes the extensions made to support Dask arrays.
 
-The simplest way to split one or more Dask arrays is with :func:`dask_ml.model_selection.train_test_split`.
+The simplest way to split one or more Dask arrays is with :func:`dask_ml.model_selection.train_test_split`:
+
+.. autosummary::
+   dask_ml.model_selection.train_test_split
```

Comment:
Maybe not necessary? We'll have a link to the full page just above.

Reply:
I wanted to keep the consistency of linking to Dask-ML documentation with `autosummary`.

Reply:
Maybe that's better suited for a separate PR. I'll revert this change.

Here's a page with the "consistency" envisioned: https://ml.dask.org/clustering.html.

Here are other pages with links to estimators without autosummaries:

* `dask_ml.preprocessing.OneHotEncoder` on https://ml.dask.org/preprocessing.html
* `dask_ml.model_selection.train_test_split()` on https://ml.dask.org/cross_validation.html
* `dask_ml.compose.ColumnTransformer` on https://ml.dask.org/compose.html
* `wrappers.ParallelPostFit` on https://ml.dask.org/meta-estimators.html